### PR TITLE
btap request

### PR DESCRIPTION
**Pull request by maybuksho**

### CODE REVIEW

File: btap/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#FROM python:3.7
+# FROM python:3.8-slim
+FROM ubuntu:20.04
+
+RUN apt-get -y update && apt-get install -y python3 python3-pip
```
Comment (blairdrummond) [inline btap/Dockerfile:5]: @sampasha See the note here about removing the apt-cache after you're done installing

https://runnable.com/blog/9-common-dockerfile-mistakes

File: btap/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+COPY download_data.py /btap
+
+
+# removal due to security risk
+#RUN apt-get remove -y libc-bin 
+
+#COPY download_data.py download_data.py
+RUN chmod a+x download_data.py
+
+#ENTRYPOINT ["python", "download_data.py"]
```
Comment (blairdrummond) [inline btap/Dockerfile:20]: @maybuksho  @sampasha  you probably want to set a `CMD` command for the image?

**Review by blairdrummond — CHANGES_REQUESTED** [review]
A few recommended changes. Also, importantly, @sampasha  this image runs as root, which is no bueno. Can you add a non-root user (say with UID 1000) and then make sure that the `btap` folder and the python script are `chown`d to the right user?

Example: https://github.com/StatCan/daaas-containers/blob/master/sftp-pull-base/Dockerfile

(There is a little "do as I say, not as I do" here, as I didn't clean the apk cache)

File: btap/Dockerfile (hunk 1 above)
Comment (maybuksho) [reply btap/Dockerfile]: @blairdrummond This is resolved, please check.

File: btap/Dockerfile (hunk 2 above)
Comment (maybuksho) [reply btap/Dockerfile]: @blairdrummond This is resolved, please check.

**Review by blairdrummond — APPROVED** [review]
LGTM!

**Pull request merged**